bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/evaluation/db/EvaluationSQLDao.kt

```diff
@@ -0,0 +1,13 @@
+package io.bucketeer.sdk.android.internal.evaluation.db
+
+import io.bucketeer.sdk.android.internal.model.Evaluation
+
+// EvaluationSQLDao interface with each method for a single purpose.
+// If you want compose multiple query or need using transaction
+// Please use the `startTransaction` method
+internal interface EvaluationSQLDao {
```

Comment:
We should also rename the event counterpart.

Reply:
Could you please tell me more details? I may not understand your suggestion clearly.

Reply:
I mean we should rename EventDao to EventSQLDao.
Also, we might want to add EventStorage just for consistency.

Reply:
thank you, I got it

Reply:
EventStorage will be done in another PR

Reply:
changed https://github.com/bucketeer-io/android-client-sdk/pull/89/commits/9d82b92d319042e062f1c4c1b993feca65873716